### PR TITLE
Update-webhooks-types

### PR DESCRIPTION
**Pull request by Virgil993**

### CODE REVIEW

File: docs/features/http-methods-webhooks.md [hunk 1]
```diff
@@ -29,22 +29,39 @@ Decorators are only supported in TypeScript, JavaScript and Go. If you are using
```
Comment (costinsin) [inline docs/features/http-methods-webhooks.md:29]: You forgot to remove the imports from `@genezio/types`.

File: docs/features/http-methods-webhooks.md [hunk 2]
```diff
@@ -29,22 +29,39 @@ Decorators are only supported in TypeScript, JavaScript and Go. If you are using
...
 
-            return response;
         }
     }
-    ```
+
+````
+
+    <Admonition type="info">
+      For TypeScript projects the expected type for the request and response objects are the standard `Request` and `Response` objects provided by the @types/node library. To use these types in your project, you need to install the `@types/node` package by running the following command:
```
Comment (costinsin) [inline docs/features/http-methods-webhooks.md:56]:
```suggestion
      For TypeScript projects the expected type for the request and response objects are the standard `Request` and `Response` objects provided by the `@types/node` library. To use these types in your project, you need to install the `@types/node` package by running the following command:
```

File: docs/features/http-methods-webhooks.md [hunk 3]
```diff
@@ -139,34 +158,18 @@ HTTP Methods Deployed:
...
 
-#### Properties
+    To learn more about these types, you can check out the Fetch API documentation for the [Request](https://developer.mozilla.org/en-US/docs/Web/API/Request) and [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response) objects.
 
-- **body - required:** An object that represents the response's body. The type of this variable can be Object, String, or Buffer.
-- **headers - optional:** A dictionary that contains the headers.
-- **statusCode - required:** The status code of the response.
-- **isBase64Encoded - optional:** This flag can be set to `true` or `false` to indicate if the `body` is base64 encoded. This flag is optional and can be omitted.&#x20;
+  </TabItem>
+</Tabs>
```
Comment (costinsin) [inline docs/features/http-methods-webhooks.md:172]: Isn't this section duplicated with the `info` Admonition above? I would remove it entirely.

**Review by costinsin — CHANGES_REQUESTED** [review]

**Comment by andreia-oca** [comment]
This is not active anymore

**Pull request closed without merging**